### DISCUSSION TITLE
Karavan support Required on the application build up-(Will provide the donation for the support of the karavan application???)

### DISCUSSION THREAD

**Discussion by kondetiramanakumar** [apache/camel-karavan#1229] [General]
@mgubaidullin  Any body in the contributors list will support the karavan application in case of the donations from our end???????

**Comment by davsclaus** [comment]
You can find a list of commercial companies that support Apache Camel.
However for Karavan specific, then AFAIK then there is the founder of this project Marat, and his company Talisman, that IMHO would be best suited for support and help (commercial).

All ASF projects are however free and volunteer based, so you cannot demand anybody to help or work for you. For that commercial companies can fill that gap.

https://camel.apache.org/manual/commercial-camel-offerings.html
